src/app/routes.py

```diff
@@ -45,12 +45,17 @@ def fluff_results():
 
     dialect = request.args["dialect"]
     linted = lint(sql, dialect=dialect)
+    fixed_sql = ""
+
+    # Only fix if linting isn't completely broken
+    if "Unrecoverable failure" not in str(linted):
```

Comment:
Is there a risk that the user's actual SQL input contains the sentinel string, `Unrecoverable failure`? Is there potentially a more precise way to check for failure, such as catching an exception, etc.?

Reply:
Yeah bit lazy of me. Fixed.